src/topics/create.js

```diff
@@ -290,6 +293,13 @@ module.exports = function(Topics) {
 		callback();
 	}
 
+	function checkTagsLength(tags, callback) {
+		if (!tags || tags.length < parseInt(meta.config.minimumTagsPerTopic, 10)) {
+			return callback(new Error('[[error:not-enough-tags, ' + meta.config.minimumTagsPerTopic + ']]'));
+		}
+		callback();
+	}
+
```

Comment:
Might as well check for tags.length > meta.config.maximumTagsPerTopic here as well.